src/models/db/account.js

```diff
@@ -10,6 +11,24 @@ const EntryGroup = require('./entry-group').EntryGroup
 const Sequelize = require('sequelize')
 const sequelize = require('../../services/db')
 
+function hashPassword (password) {
+  // cache hashes of passwords used in tests to speed up tests 4X
```

Comment:
No, production code does not have access to test code. We could move it to a separate file in the production code, but I'm not sure if it's worth it since it's pretty small.